backbone/src/main/res/values/dimens.xml

```diff
@@ -11,6 +11,9 @@
     <dimen name="rsb_item_size_default">@dimen/rsb_item_size_small</dimen>
     <dimen name="rsb_item_size_divider">1dp</dimen>
     <!-- Item Margin -->
+    <dimen name="rsb_margin_top_extra_small">12dp</dimen>
+    <dimen name="rsb_margin_top_small">14dp</dimen>
```

Comment:
I create this because there is : 
<dimen name="rsb_margin_top_extra_small">12dp</dimen>
    <dimen name="rsb_margin_top_small">14dp</dimen>
    <dimen name="rsb_margin_top_large">20dp</dimen>

if we call the small medium and want to add something like for 16dp i think we want to have the medium always available :)